### PR TITLE
Added demo and code examples to the phone field documentation

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
#48

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-phone-field-demo-and-code-example](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-phone-field-demo-and-code-example)

**Review by Blackbaud-DeniseCisneros — DISMISSED** [review]

**Review by blackbaud-johnly — DISMISSED** [review]

File: src/app/docs/phone-field/phone-field-docs.component.ts [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+  styleUrls: ['./phone-field-docs.component.scss']
 })
-export class PhoneFieldDocsComponent { }
+export class PhoneFieldDocsComponent implements OnInit {
+
+  public phoneForm: FormGroup;
+
+  public phoneControl: FormControl;
+
+  public ngOnInit() {
```
Comment (Blackbaud-AlexKingman) [inline src/app/docs/phone-field/phone-field-docs.component.ts]: Needs a return type.

File: src/app/public/plugin-resources/code-examples/phone-field/basic/phone-field-demo.component.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  templateUrl: './phone-field-demo.component.html',
+  styleUrls: ['./phone-field-demo.component.scss']
+})
+export class PhoneFieldDemoComponent implements OnInit {
+
+  public phoneForm: FormGroup;
+
+  public phoneControl: FormControl;
+
+  public ngOnInit() {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/plugin-resources/code-examples/phone-field/basic/phone-field-demo.component.ts]: Needs a return type.

File: src/app/docs/phone-field/phone-field-docs.component.ts [hunk 3]
```diff
@@ -1,9 +1,29 @@
...
-  templateUrl: './phone-field-docs.component.html'
+  templateUrl: './phone-field-docs.component.html',
+  styleUrls: ['./phone-field-docs.component.scss']
 })
-export class PhoneFieldDocsComponent { }
+export class PhoneFieldDocsComponent implements OnInit {
+
+  public phoneForm: FormGroup;
+
+  public phoneControl: FormControl;
```
Comment (Blackbaud-AlexKingman) [inline src/app/docs/phone-field/phone-field-docs.component.ts:18]: Check your alphabetical ordering of properties.

File: src/app/public/plugin-resources/code-examples/phone-field/basic/phone-field-demo.component.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+@Component({
+  selector: 'app-phone-field-demo',
+  templateUrl: './phone-field-demo.component.html',
+  styleUrls: ['./phone-field-demo.component.scss']
+})
+export class PhoneFieldDemoComponent implements OnInit {
+
+  public phoneForm: FormGroup;
+
+  public phoneControl: FormControl;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/plugin-resources/code-examples/phone-field/basic/phone-field-demo.component.ts:18]: Check your alphabetical ordering of properties.

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
Just a few minor code style things. One last question - our UX guidelines show an alert when the form is invalid. Should we do that in the demo/example?

File: src/app/docs/phone-field/phone-field-docs.component.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/docs/phone-field/phone-field-docs.component.ts]: Done

File: src/app/public/plugin-resources/code-examples/phone-field/basic/phone-field-demo.component.ts (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/plugin-resources/code-examples/phone-field/basic/phone-field-demo.component.ts]: Done

File: src/app/docs/phone-field/phone-field-docs.component.ts (hunk 3 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/docs/phone-field/phone-field-docs.component.ts:18]: Done

File: src/app/public/plugin-resources/code-examples/phone-field/basic/phone-field-demo.component.ts (hunk 4 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/plugin-resources/code-examples/phone-field/basic/phone-field-demo.component.ts:18]: Done

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-AlexKingman good call on the error message. I have added this. The PR is ready for a new review.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-phone-field-demo-and-code-example](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-phone-field-demo-and-code-example)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**